cypress/integration/booked-view-test.js

```diff
@@ -0,0 +1,38 @@
+describe('Booked Jobs View', () => {
+
+  beforeEach(() => {
+    cy.modifiedJob()
+    .visit('http://localhost:3000')
+  });
+
+  describe('Booked Jobs User Story', () => {
+
+    it('Should show booked jobs only when View Booked button is clicked', () => {
```

Comment:
It might be good to rephrase this line. I'm currently reading it as you shouldn't be able to booked jobs until the book jobs button is pushed.

Reply:
Good catch! Revised wording here.